source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -1015,8 +1015,20 @@ export class TextBlock extends BaseTextBlock {
                 element.style.wordWrap = "break-word";
 
                 if (this.maxLines && this.maxLines > 0) {
-                    element.style.maxHeight = (this._computedLineHeight * this.maxLines) + "px";
                     element.style.overflow = "hidden";
+
+                    if (Utils.isInternetExplorer() || !GlobalSettings.useWebkitLineClamp) {
+                        element.style.maxHeight = (this._computedLineHeight * this.maxLines) + "px";
+                    }
+                    else {
+                        // While non standard, --webkit-line-clamp works in every browser (except IE)
+                        // and is a great solution to support the maxLines feature with ellipsis
+                        // truncation. With --webkit-line-clamp there is need to use explicit line heights
+                        element.style.removeProperty("line-height");
+                        element.style.display = "-webkit-box";
+                        element.style.webkitBoxOrient = "vertical";
```

Comment:
looks like box orient [doesn't work by default](https://caniuse.com/?search=-webkit-box-orient) in Firefox? is that something we should be concerned about?

Reply:
I've tested the change in Edge, Chrome, Firefox, Opera and Brave and it worked across the board.

Reply:
Ah - and now I remember what happened when I looked at the DOM in Firefox. It actually automatically replaced `--webkit-box-orient` with `-moz-box-orient`:
![image](https://user-images.githubusercontent.com/1334689/105437688-5df05700-5c16-11eb-8213-cbda23edb7d7.png)

So the question is, how did that happen?
- Is that something new in more recent versions of Firefox?
- Is it TypeScript magic?

I'm going to assume the latter, but I can't find any info.